encoding/resp/resp.go

```diff
@@ -119,6 +122,22 @@ func (r *Encoder) Array(size int) error {
 	return err
 }
 
+func (r *Encoder) BulkStrings(strs [][]byte) error {
```

Comment:
BulkStrings can be combined with BulkString and Array. RESP should implement the primitive data types only.